02-clients/01-studio.md

```diff
@@ -0,0 +1,421 @@
+- [**Run Query**] (play (▶) icon) button — run query or queries in the active tab of the Text editor panel. It will 
+  open a transaction (with the session type and transaction type specified in the Toolbar on the left) and execute 
+  the TypeQL code against the selected database.
+- [**Stop Signal**] (lighting (⚡) icon) button — stop query execution after the next result.
+
+##### Run query
+
+Make sure TypeDB Studio [connected](#connect-to-typedb-server) to a TypeDB Server, 
+database [selected](#select-active-database) and Project directory [opened](#open-a-project-directory).
+
+1. [Write a new](#create-a-new-file) or [open an existing](#open-an-existing-file) file with query to run.
+
+   ![Query text](../images/studio/project-schema-pasted.png)
+
+2. Choose the correct session (`schema`/`data`) type and transaction (`read`/`write`) type to run the query by  
+   clicking on those options on the Toolbar switches.
+
+3. Run query by clicking on the [**Run Query**] (▶) button. 
+
+   ![Write the Schema](../images/studio/project-schema-query-run.png)
+
+4. If it's the `write` transaction type — wait for transaction control options to be ready and commit or close the 
+   transaction (by clicking on the [**Commit transaction**] (✔) or [**Close transaction**] (x) buttons respectively).
+
+The result of the query is displayed in the Run panel under the Text editor.
+
+If any error occurs during the query validation or transaction it is shown as red popup in the bottom right corner of 
+the TypeDB Studio window. Additional information about the error is displayed in the results of the query in the 
+Run panel.
+
+If a `write` transaction committed successfully — a blue pop-up with a transaction confirmation is displayed. See 
+example below.
+
+![Commit the Transaction](../images/studio/project-schema-committed.png)
+
+## Types explorer capabilities
+
+Under the Project panel that shows the Project directory TypeDB has Types panel that shows all types of the schema of 
+the current database. 
+
+<div class="note">
+[Note]
+If the Types panel is not displayed — click the Types vertical tab on the left edge of the TypeDB Studio window.
+</div>
+
+All types are divided in three top level [base types](../0001-typedb/02-dev/02-schema.md#types):
+
+- Attribute
+- Entity
+- Relation
+
+![Types explorer](../images/studio/types-explorer.png)
+
+All types displayed in the exact hierarchy they were created (subtypes inside their supertypes). We can easily 
+collapse or expand all the types in the structure tree by clicking the associated buttons on the top of the Types panel 
+with arrows pointing down or up respectively.
+
+Types panel can be closed by clicking the X icon in the top right corner of the panel. To reopen the panel click the 
+Types tab on the left edge of the TypeDB Studio window.
+
+### Export schema
+
+We can export schema of the database by clicking the [**Export Schema**] button (second in row from left to right) on 
```

Comment:
The button is labeled "Export Schema Types" to indicate that it does not include rules. The warning below is still valid as it's annoying.

Reply:
Was the third word of the button named added in 2.17? Because in older version of Studio (when the text was written) I do remember it wasn't there.